development-vm/README.md

```diff
@@ -1,276 +1,212 @@
-# Developing on GOV.UK Virtual Machines
+## Welcome to GOV.UK.
```

Comment:
I can't see this in context, but it looks like an H2 following straight on from the H1 of the page title. We tend not to do one H right after another, style-wise. I'm not sure it's needed, as we repeat the same words right underneath...

If we do feel it's essential, let's remove the full stop at the end – again, it's against style